### ISSUE TITLE
Reorganize Scene Tab

### ISSUE THREAD

**Issue by sgpearse** [NCAR/VAPOR#3058]
The Scene tab (next to the Renderers tab) is very busy, containing the already complex Annotation, Animation, and Viewpoint tabs.  Often the first thing I do in a new session is to stretch the Z axis, but it's buried under Scene->Viewpoint->Transform.  This is hard to find, being one of the first things a user will need to do after loading data.

I think this section would be greatly simplified by doing the following things.

1) Add a time step slider to the top-level controls, as well as an indicator for how many time steps are in the current dataset.
2) Move the Animation controls to the Tools menu, alongside the Plot, Statistics, and Python editors.
3) Change the Annotation controls from a tab into a Renderer object, which can have multiple instances.
4) Move the Viewpoint tab next to the Renderers Tab.

Mockup of proposed changes:

![image](https://user-images.githubusercontent.com/9522770/156195336-eb553235-04f0-40cc-88d4-abd9213271d1.png)

**Comment by clyne** [comment]
@sgpearse is there a way to make incremental progress on the usability without committing to such a significant change, such as adding scale factors on the title bar?

**Comment by sgpearse** [comment]
@clyne Yes, that would make stretching easier.   What do you think about adding the max time-step number as well?  It's not always obvious to me where I am in a given time series.
